libraries/botbuilder-storage-mssql/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@botbuildercommunity/storage-mssql",
+  "version": "0.1.0",
```

Comment:
1.0.0? Looks like the package meets all the quality standards. ;-)

Reply:
:smile:

Reply:
Fixed.